packages/server/modules/workspaces/graph/resolvers/workspaces.ts

```diff
@@ -761,6 +762,18 @@ export = FF_WORKSPACES_MODULE_ENABLED
         },
         role: async (parent) => {
           return parent.workspaceRole
+        },
+        projectRoles: async (parent, _args, ctx) => {
+          const projectRoles = await getRolesByUserIdFactory({ db })({
+            userId: parent.id,
+            workspaceId: parent.workspaceId
+          })
+          return await Promise.all(
```

Comment:
The actual getStream.load() call should preferably be made in the ProjectRole.project resolver - otherwise we're prefetching all of these streams, even tho the requester might only be interested in roles

Reply:
This Promise.all() is no longer needed here